package.json

```diff
@@ -52,12 +52,13 @@
         "eslint-plugin-jsx-a11y": "^6.2.1",
         "eslint-plugin-prettier": "^3.1.0",
         "eslint-plugin-react": "^7.13.0",
-        "eslint-config-react-app": "^4.0.1",    
+        "eslint-config-react-app": "^4.0.1",
         "eslint-plugin-react-hooks": "^1.6.0",
         "express": "~4.16.3",
         "full-icu": "~1.2.1",
         "husky": "^2.3.0",
         "jest": "^23.6.0",
+        "jest-dom": "3.5.0",
```

Comment:
I have no other idea hot to test whether a dom element (for menu item) is disabled. I've picked the old version with the purpose of having the smallest changes to `yarn.lock`